src/client/styles/fonts/_frutiger.scss

```diff
@@ -0,0 +1,7 @@
+@import url("https://fast.fonts.net/cssapi/0985eb4d-eec4-488b-baa4-22fe85c5ce32.css");
+body p, h1, h2, h3, details summary, ul {
```

Comment:
I guess to be honest, So long as it works, this is only really a stop gap until the new library is released. I'm happy to approve the PR, with the caveat that you will need to switch to the new Library when available.